infra/modules/azuread-local-cert/variables.tf

```
@@ -10,9 +10,9 @@ variable "cert_expiration_days" {
   description = "cert will expire in this many days"
 }
 
-variable "application_object_id" {
+variable "application_id" {
   type        = string
-  description = "object ID of the Azure AD application to authenticate"
+  description = "application ID of the Azure AD application to authenticate; this is NOT the object ID of the application"
```

Just to match with description appeared in portal
```suggestion
  description = "application (client) ID of the Azure AD application to authenticate; this is NOT the object ID of the application"
```